Database.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,7 @@
 		2A0CA5751B294C9200ED8781 /* libsqlite3.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 2A0CA5741B294C9200ED8781 /* libsqlite3.dylib */; };
```

Comment:
Is there gonna be a problem if we rename the project? Just think that the name 'database' is not very suitable